.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
+  - repo: https://github.com/ambv/black
+    rev: stable
+    hooks:
+    - id: black
+default_language_version:
+  python: python3.7
```

Comment:
the installation instructions in the README currently suggest python3.6
I guess we should change one of the two

Reply:
I propose we move to python3.7. There are [several excellent additions and improvements over 3.6](https://docs.python.org/3/whatsnew/3.7.html). Several relate specifically to the `typing` module and type hinting, e.g. "the speed of various operations with types increased up to 7 times", which will impact this project positively. I've updated the PR to reflect this.